### ISSUE TITLE
prefix hbs helper

### ISSUE THREAD

**Issue by alxbenz** [frontend-framework/build-framework#27]
From using the build-framework in an actual project we found that the naming of the hbs helpers collide with variable names. When sharing the name with a helper the variables don't get rendered.

This is problematic for example with the text helper and variables names text.

We should discuss how to handle this problem.

I propose two things:
* prefixing all handlebars helpers (for example "hText")
* prefixing variables with the component name that they belong to (for example text for an alert component should be named "alertText")

**Comment by janrembold** [comment]
I like the idea of prefixing variable names. This will help especially in more complex components.
But I would suggest to prefix the helpers with a fully qualified name like e.g. helperText().

**Comment by bennyvi** [comment]
I think it would make sense to do both, prefixing all handlebars helpers with helperForSomething(). and also prefixing variables with the component name.

**Comment by timomayer** [comment]
I agree too. We should prefix both helpers (helper) prefix and prefix component variables with component name

**Comment by dweiger** [comment]
I also like the idea. What about the prefix "bio". So we get bio-text or bioText? Wich syntax would you prefer?

**Comment by smohadjer** [comment]
@timomayer Helpers are prefixed now.
